lib/engine/game/g_1880/game.rb

```diff
@@ -16,14 +16,24 @@ class Game < Game::Base
 
         TRACK_RESTRICTION = :permissive
         SELL_BUY_ORDER = :sell_buy
+        SELL_MOVEMENT = :down_per_10
+        MARKET_SHARE_LIMIT = 100
```

Comment:
Is MARKET_SHARE_LIMITED needed if shares go back to the corporation?

Reply:
not needed. will clean